---
title: Publications
---

## Representatives:
- **Song W**, Zhang S, Perez M, Li J, Ma H, Thomas T,  Qiu J, Qian P, 2025, 'Distinct patterns of microbial association across deep-sea corals from the Western Pacific Magellan Seamounts', Microbiology Spectrum. (IF: **3.8**) [https://doi.org/10.1128/spectrum.02093-25](https://doi.org/10.1128/spectrum.02093-25) 
- Zhang S; **Song W**; Marinos G; Waschina S; Zimmermann J; Kaleta C; Thomas, T, 2024, Genome-scale metabolic modelling reveals interactions and key roles of symbiont clades in a sponge holobiont. Nature Communications, 15(1), 10858. (IF: **14.7**) [https://doi.org/10.1038/s41467-024-55222-W](https://doi.org/10.1038/s41467-024-55222-W)
- Wang W; **Song W**; Majzoub M; Feng X; Xu B; Tao J; Fan L, 2024, Decoupling of strain-and intrastrain-level interactions of microbiomes in a sponge holobiont. Nature Communications, 15(1), 8205. (IF: **14.7**) [https://doi.org/10.1038/s41467-024-52464-6](https://doi.org/10.1038/s41467-024-52464-6)
- **Song W**; Zhang S; Majzoub ME; Egan S; Kjelleberg S; Thomas T, 2023, 'The impact of interspecific competition on the genomic evolution of Phaeobacter inhibens and Pseudoalteromonas tunicata during biofilm growth', Environmental Microbiology, 2023 Dec 7. (IF: **5.476**) [https://doi.org/10.1111/1462-2920.16553](https://doi.org/10.1111/1462-2920.16553)
- **Song W**, Zhang S, Thomas T, 2022, 'MarkerMAG: linking metagenome-assembled genomes (MAGs) with 16S rRNA marker genes using paired-end short reads', Bioinformatics, btac398. (IF: **6.931**) [https://doi.org/10.1093/bioinformatics/btac398](https://doi.org/10.1093/bioinformatics/btac398)
- **Song W**; Wemheuer B; Steinberg PD; Marzinelli EM; Thomas T, 2021, 'Contribution of horizontal gene transfer to the functionality of microbial biofilm on a macroalgae', ISME Journal, vol. 15, pp. 807 – 817. (IF: **13.214**) [https://doi.org/10.1038/s41396-020-00815-8](https://doi.org/10.1038/s41396-020-00815-8)
- **Song W**; Wemheuer B; Zhang S; Steensen K; Thomas T, 2019, 'MetaCHIP: Community-level horizontal gene transfer identification through the combination of best-match and phylogenetic approaches', Microbiome, vol. 7, pp. 36. (IF: **16.837**) [https://doi.org/10.1186/s40168-019-0649-y](https://doi.org/10.1186/s40168-019-0649-y)
- **Song W**; Thomas T, 2017, 'Binning-refiner: Improving genome bins through the combination of different binning programs', Bioinformatics, vol. 33, pp. 1873 – 1875. (IF: **6.931**) [https://doi.org/10.1093/bioinformatics/btx086](https://doi.org/10.1093/bioinformatics/btx086)

## Complete list:
- Li J, Del Olmo G, Zhang S, Philip C, Saha M, **Song W**, Egan S, Wood G, 2025, 'Host-Microbiome Ecology in a Seaweed Holobiont: Current Understanding, Technological Advancements, and Manipulation Approaches', Technological Advancements, and Manipulation Approaches. [http://dx.doi.org/10.2139/ssrn.5391363](http://dx.doi.org/10.2139/ssrn.5391363)
- Li Z, Zhao C, Mao Z, Zhang F, Dong L, Song C, Chen Y, Fu X, Ao Z, Xiong Y, Hui Q, **Song W**, Penttinen P, Zhang S, 2025, 'Structure and metabolic function of spatiotemporal pit mud microbiome. Environmental microbiome', 20;20(1):10. [https://doi.org/10.1186/s40793-025-00668-8](https://doi.org/10.1186/s40793-025-00668-8)
- Khan T, **Song W**, Nappi J, Marzinelli E, Egan S, Thomas T, 2023, 'Functional guilds and drivers of diversity in seaweed-associated bacteria', FEMS Microbes, Dec 14:xtad023. [https://doi.org/10.1093/femsmc/xtad023](https://doi.org/10.1093/femsmc/xtad023)
- Fu S, Zhang Y, Wang R, Qiu Z, **Song W**, Yang Q, Shen L, 2023, 'A novel culture-enriched metagenomic sequencing strategy effectively guarantee the microbial safety of drinking water by uncovering the low abundance pathogens', Journal of Environmental Management, 345, 118737. (IF: 8.91) [https://doi.org/10.1016/j.jenvman.2023.118737](https://doi.org/10.1016/j.jenvman.2023.118737)
- Chen L, Hong T, Wu Z, **Song W**, Chen S, Liu Y, Shen L, 2023, 'Genomic analyses reveal a low temperature adapted clade in Halorubrum, a widespread haloarchaeon across global hypersaline environments', BMC Genomics, 24, 508. (IF: 4.558) [https://doi.org/10.1186/s12864-023-09597-7](https://doi.org/10.1186/s12864-023-09597-7)
- Nguyen VH, Wemheuer B, **Song W**, Bennett H, Webster NS, Thomas T, 2023, 'Identification, classification, and functional characterization of novel sponge-associated acidimicrobiial species', Systematic and Applied Microbiology, 126426. (IF: 4.064) [https://doi.org/10.1016/j.syapm.2023.126426](https://doi.org/10.1016/j.syapm.2023.126426)
- Nguyen VH, Wemheuer B, **Song W**, Bennett H, Palladino G, Burgsdorf I, Thomas T, 2023, 'Functional characterization and taxonomic classification of novel gammaproteobacterial diversity in sponges', Systematic and Applied Microbiology, 46(2), 126401. (IF: 4.064) [https://doi.org/10.1016/j.syapm.2023.126401](https://doi.org/10.1016/j.syapm.2023.126401)
- Zhang S; **Song W**; Nothiasc L; Couvillion S; Webster N; Thomas T, 2022, 'Comparative metabolomic analysis reveals shared and unique chemical interactions in sponge holobionts', Microbiome, 10, 22. (IF: **16.837**) [https://doi.org/10.1186/s40168-021-01220-9](https://doi.org/10.1186/s40168-021-01220-9)
- Garritano AN, **Song W**, Thomas T,2022, 'Carbon fixation pathways across the bacterial and archaeal tree of life', PNAS Nexus. 2022 Oct 4. (IF: NA) [https://doi.org/10.1093/pnasnexus/pgac226](https://doi.org/10.1093/pnasnexus/pgac226)
- Liu R, Wei X, **Song W**, Wang L, Cao J, Wu J, Thomas T, Jin T, Wang Z, Wei W, Wei Y, 2022, 'Novel Chloroflexi genomes from the deepest ocean reveal metabolic strategies for the adaptation to deep-sea habitats', Microbiome,10(1):1-7. (IF: 16.837) [https://doi.org/10.1186/s40168-022-01263-6](https://doi.org/10.1186/s40168-022-01263-6)
- Robbins SJ; **Song W**; Engelberts JP; Glasl B; Slaby BM; Boyd J; Marangon E; Botté ES; Laffy P; Thomas T; Webster NS, 2021, 'A genomic view of the microbiome of coral reef demosponges', ISME Journal, vol. 15, pp. 1641 – 1654. (IF: **13.214**) [https://doi.org/10.1038/s41396-020-00876-9](https://doi.org/10.1038/s41396-020-00876-9)
- Dong X; Zhang C; Li W; Weng S; **Song W**; Li J; Wang Y, 2021, 'Functional diversity of microbial communities in inactive seafloor sulfide deposits', FEMS Microbiology Ecology, Aug;97(8): fiab108. (IF: 4.519) [https://doi.org/10.1093/femsec/fiab108](https://doi.org/10.1093/femsec/fiab108)
- Liu Q; **Song W**; Zhou YG; Dong XZ; Xin YH, 2020, 'Phenotypic divergence of thermotolerance: Molecular basis and cold adaptive evolution related to intrinsic DNA flexibility of glacier-inhabiting Cryobacterium strains', Environmental Microbiology, vol. 22, pp. 1409 – 1420. (IF: 5.476) [https://doi.org/10.1111/1462-2920.14957](https://doi.org/10.1111/1462-2920.14957)
- Zhang S; **Song W**; Wemheuer B; Reveillaud J; Webster N; Thomas T, 2019, 'Comparative genomics reveals ecological and evolutionary insights into sponge associated Thaumarchaeota', mSystems, vol. 4. (IF: 7.324) [https://doi.org/10.1128/msystems.00288-19](https://doi.org/10.1128/msystems.00288-19)
- **Song W**; Thomas T; Edwards RJ, 2019, 'Complete genome sequences of pooled genomic DNA from 10 marine bacteria using PacBio long-read sequencing', Marine Genomics, vol. 48. (IF: 2.143) [https://doi.org/10.1016/j.margen.2019.05.002](https://doi.org/10.1016/j.margen.2019.05.002)
- Zhang S; **Song W**; Yu M; Lin X, 2017, 'Comparative genomics analysis of five Psychrobacter strains isolated from world-wide habitats reveal high intra-genus variations', Extremophiles, vol. 21, pp. 581 – 589. (IF: 3.035) [https://doi.org/10.1007/s00792-017-0927-1](https://doi.org/10.1007/s00792-017-0927-1)
- **Song W**; Steensen K; Thomas T, 2017, 'HgtSIM: A simulator for horizontal gene transfer (HGT) in microbial communities', PeerJ, vol. 2017. (IF: 3.061) [https://doi.org/10.7717/peerj.4015](https://doi.org/10.7717/peerj.4015)
- Che S; Song L; **Song W**; Yang M; Liu G; Lin X, 2013, 'Complete Genome Sequence of Antarctic Bacterium Psychrobacter sp. Strain G.', Genome Announc, vol. 1. [https://doi.org/10.1128/genomea.00725-13](https://doi.org/10.1128/genomea.00725-13)
- **Song W**; Lin X; Che S, 2013, 'Identification of regulatory sequences and expression analysis of OmpR gene under different stress conditions in the antarctic bacterium Psychrobacter sp. G.', Curr Microbiol, vol. 66, pp. 259 – 265. (IF: 2.408) [https://doi.org/10.1007/s00284-012-0266-5](https://doi.org/10.1007/s00284-012-0266-5)
- Che S; **Song W**; Lin X, 2013, 'Response of heat-shock protein (HSP) genes to temperature and salinity stress in the antarctic psychrotrophic bacterium Psychrobacter sp. G.', Curr Microbiol, vol. 67, pp. 601 – 608. (IF: 2.408) [https://doi.org/10.1007/s00284-013-0409-3](https://doi.org/10.1007/s00284-013-0409-3)
- **Song W**; Lin X; Huang X, 2012, 'Characterization and expression analysis of three cold shock protein (CSP) genes under different stress conditions in the Antarctic bacterium Psychrobacter sp. G', Polar Biology, vol. 35, pp. 1515 – 1524. (IF: 2.493) [https://doi.org/10.1007/s00300-012-1191-6](https://doi.org/10.1007/s00300-012-1191-6)
